src/components/staffing/ProjectCell.js

```diff
@@ -12,7 +12,8 @@ export default class ProjectCell extends React.Component {
 
   render() {
     const { rowIndex, field, data, ...props } = this.props
-    const currentWeek = Object.keys(data[rowIndex].staffing).sort()[0]
+    const weeks = Object.keys(data[rowIndex].staffing)
+    const currentWeek = [...weeks].sort()[0]
```

Comment:
It's better but the part that is important to me to fix is to avoid `.sort()[0]` → because it's misleading and can confuse devs (they might wonder if there is a bug since `sort` does the changes to the input array)

What I would suggest here is:
```js
const sortedWeeks = Object.keys(data[rowIndex].staffing).sort()
const currentWeek = sortedWeeks[0]
```

Reply:
PS: you could even create a little util function `getCurrentWeek` to extract this logic in it (or getSortedWeeks if you need to reuse the weeks further in the component)

Reply:
Fixed!